build/tasks.js

```diff
@@ -85,7 +86,22 @@ export default {
   },
 
   async test(args = []) {
-    await this.build({ test: true });
+    // When testing locally, just use whatever build already existed,
+    // regardless of whether it's production or development.
+    const { development } = BuildUtils.safeGetBuildConfig();
+    logger.info(`Attempting to use existing ${development ? 'dev' : 'production'} build...`);
+    await this.build({ development });
+    await Lazy.test(args);
+  },
+
+  async testCI(args = []) {
+    // These tests run on the CI server.
+    logger.info('Making a production build...');
+    await this.build();
+    await Lazy.test(args);
+
+    logger.info('Making a development build...');
+    await this.build({ development: true });
```

Comment:
I think testing both dev and production builds is unnecessary, and increases our chances of webdriver intermittents as well, but whatever